src/Microsoft.Extensions.Configuration.UserSecrets/build/netstandard1.0/Microsoft.Extensions.Configuration.UserSecrets.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(UserSecretsAssemblyAttributeFile)' == '' and $(MSBuildProjectFile.EndsWith('.vbproj')">
```

Comment:
Is this really the only way to detect the project language? That kinda sucks...

If it's the only way though, so be it :)